pandas/io/parsers.py

```diff
@@ -2327,14 +2327,21 @@ def _next_line(self):
                 try:
                     orig_line = next(self.data)
                 except csv.Error as e:
+                    msg = str(e)
+
                     if 'NULL byte' in str(e):
-                        raise csv.Error(
-                            'NULL byte detected. This byte '
-                            'cannot be processed in Python\'s '
-                            'native csv library at the moment, '
-                            'so please pass in engine=\'c\' instead.')
-                    else:
-                        raise
+                        msg = ('NULL byte detected. This byte '
+                               'cannot be processed in Python\'s '
+                               'native csv library at the moment, '
+                               'so please pass in engine=\'c\' instead')
+
+                    if self.skipfooter > 0:
+                        reason = ('Error could possibly be due to '
+                                  'skipfooter being ignored in Python\'s '
+                                  'native csv library.')
```

Comment:
I am wondering if we can word this a little different. As now it seems to imply that `skipfooter` is ignored altogether ("so why can I specify it then"). 
Do I understand it correctly that the skipped lines are still parsed, but not used when converting the parsed lines to a DataFrame (so eg the number of values in the skipped lines do not need to match the rest of the file)?